.github/workflows/image-on-tag.yml

```diff
@@ -0,0 +1,54 @@
+name: Build Image on Tag Push
+
+on:
+  push:
+    tags:
```

Comment:
What if it will be need to check image for some tests? Maybe it make sense add portability run flow by hands

Reply:
Could we add it on demand if will be needed ?

Reply:
Sure!